libs/application/template-api-modules/src/lib/modules/templates/parental-leave/parental-leave.service.ts

```diff
@@ -189,6 +189,27 @@ export class ParentalLeaveService extends BaseTemplateApiService {
     }
   }
 
+  async setBirthDate({ application }: TemplateApiModuleActionProps) {
+    const { applicationFundId } = getApplicationExternalData(
+      application.externalData,
+    )
+    if (applicationFundId) {
```

Comment:
If applicationFundId is found then we get the birth date?

Reply:
I think this was a brain fart. Should not be there.